script/script.js

```diff
@@ -0,0 +1,22 @@
+let name;
+let email;
+let message;
+function submitAlert(){
+   name = document.getElementById('name').value;
+   email = document.getElementById('email').value;
+   message = document.getElementById('message').value;
+   alert(`You are about to send an email.\n\nName: ${name}.\nE-mail: ${email}.\nMessage: ${message}`);
+}
+function resetInput(){
+    document.getElementById("form-main").reset();
+}
+   
```

Comment:
Great job sorting out some DOM scripting before Bob has gone over it in class at all for the form submission alert!